bundles/orbisgis-omanager/src/main/java/org/orbisgis/omanager/ui/ActionBundleFactory.java

```diff
@@ -82,6 +86,11 @@ public ActionBundleFactory(BundleContext bundleContext,Component frame, boolean
         this.bundleContext = bundleContext;
         this.frame = frame;
         this.warnUser =warnUser;
+
+        ServiceReference<ExecutorService> executorServiceReference = bundleContext.getServiceReference(ExecutorService.class);
```

Comment:
Do not use bundleContext.getServiceReference (use it on Groovy, but never in Java bundles). 

Add a service reference in ManagerMenuFactory then add executor service in constructor of class that requires it. 

As a side note, you can also make this service dependency optional like this:
https://github.com/orbisgis/orbisgis/blob/master/bundles/geocatalog/src/main/java/org/orbisgis/geocatalog/impl/Catalog.java#L139